src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -473,14 +471,14 @@ protected function _bundleTranslatedFields($entity)
         }
 
         $results = $this->_findExistingTranslations($find);
-        $model = $this->config('conditions.model');
+        $alias = $this->_table->alias();
 
```

Comment:
I'm saying that when `config['model']` was added (https://github.com/cakephp/cakephp/commit/2f16e0c8) it was meant to be the actual value of `model` in the DB for the translated records. Someone needed it to be customizable instead of using the table alias.